src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -0,0 +1,42 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--<Import Project="Microsoft.NET.RuntimeIdentifierInference.targets" />-->
+
+  <PropertyGroup>
+    <PublishWithAspNetCoreTargetManifest Condition="'$(PublishWithAspNetCoreTargetManifest)'==''">true</PublishWithAspNetCoreTargetManifest>
+  </PropertyGroup>
+
+<!--
+*****************************************************************************
+Target: PublishWithAspNetCoreTargetManifest
+Append the default ASP.NET Core runtime package store manifest during publish
+*****************************************************************************
+-->
+  <Target
+    Name="PublishWithAspNetCoreTargetManifest"
+    BeforeTargets="Publish"
+    DependsOnTargets="PrepareForPublish"
+    Condition="'$(PublishWithAspNetCoreTargetManifest)'=='true' and '$(PublishableProject)'=='true'" >
+
+    <PropertyGroup>
+      <ResolvedRuntimeIdentifier>$(AspNetCoreTargetManifestRuntimeIdentifier)</ResolvedRuntimeIdentifier>
+      <ResolvedRuntimeIdentifier Condition="'$(AspNetCoreTargetManifestRuntimeIdentifier)'=='' and '$(OS)'=='Windows_NT' and '$(Platform)'!='x86'">win7-x64</ResolvedRuntimeIdentifier>
+      <ResolvedRuntimeIdentifier Condition="'$(AspNetCoreTargetManifestRuntimeIdentifier)'=='' and '$(OS)'=='Windows_NT' and '$(Platform)'=='x86'">win7-x86</ResolvedRuntimeIdentifier>
+      <ResolvedRuntimeIdentifier Condition="'$(AspNetCoreTargetManifestRuntimeIdentifier)'=='' and '$(OS)'=='OSX'">osx.10.12-x64</ResolvedRuntimeIdentifier>
```

Comment:
Only in MSBuild 15.3. dotnet-CLI 1.0.0 has MSBuild 15.1. Using it will require users to upgrade VS and dotnet-CLI.